docs/manual/backend-configuration/store-configuration/general-settings/notifications_overview/_index.en.md

```diff
@@ -18,7 +18,8 @@ If you imagine `Benachrichtigungszentrum` detached from Isotope eCommerce, some
 
 - A notification always has a trigger, and since that trigger should potentially trigger several different messages, a notification consists of one or more messages.
 
-{{% notice warning %}}Note the different usage of `Benachrichtigung` and a single `Nachricht`. A `Benachrichtigung` is a vessel of one or more `Nachrichten`.{{% /notice %}}
+{{% notice warning %}}Note the different usage of `Benachrichtigung` and a single `Nachricht`. A `Benachrichtigung` is a vessel of one or more `Nachrichten`.
```

Comment:
is this change nescessary?

Reply:
I got an error on this page and this fixed it somehow. I reverted it and it still builds :man_beard: